### PR TITLE
MAT-6342 verifying dependencies in madie-layout

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-6342](https://jira.cms.gov/browse/MAT-6342)
(Optional) Related Tickets:

### Summary
verifying dependencies in madie-layout

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,9 +62,7 @@
     "react-router-dom": "^5.3.0",
```
Comment (RohitKandimalla) [inline package.json]: I believe we need this in dependencies array

File: package.json [hunk 2]
```diff
@@ -28,9 +28,6 @@
     "@babel/preset-react": "^7.14.5",
     "@babel/preset-typescript": "^7.14.5",
     "@babel/runtime": "^7.14.6",
-    "@madie/madie-auth": "^0.0.2",
-    "@madie/madie-editor": "^0.0.2",
-    "@madie/madie-root": "^0.0.2",
     "@okta/okta-react": "^6.4.3",
```
Comment (RohitKandimalla) [inline package.json]: @okta/okta-react should be a PROD dep

File: package.json [hunk 3]
```diff
@@ -98,7 +96,9 @@
     "single-spa": "^5.9.3",
     "single-spa-react": "^4.4.0",
     "style-loader": "^3.3.1",
```
Comment (RohitKandimalla) [inline package.json]: style-loader can be a dev dep
@types/react-router-dom can be a dev dep

idk what qs is used for, I don't see any usage. but we are uncertain, let me be for now

Comment (sb-prateekkeerthi) [reply package.json]: thank you, I changed it

File: package.json (hunk 2 above)
Comment (sb-prateekkeerthi) [reply package.json]: thank you, I changed it

File: package.json (hunk 1 above)
Comment (sb-prateekkeerthi) [reply package.json]: thank you, I changed it

**Review by RohitKandimalla — APPROVED** [review]
